tests/test_filtering.py

```diff
@@ -36,7 +35,7 @@ def test_urgent(tmpdir, runner, create):
     create(
         'one.ics',
         'SUMMARY:haha\n'
-        'PRIORITY: 9\n'
+        'URGENT: 9\n'
```

Comment:
I have a doubt.
If priority = 9 implies urgent, aren't we wrong in keeping high as 4, medium as 5  and low as 9?
Shouldn't it be none 0, medium 5, low 4, high 9?